src/css/imports/transitions.less

```diff
@@ -0,0 +1,7 @@
+.jw-controls-right,
+.jw-controlbar,
+.jw-display,
+.jw-overlay,
+.jw-dock {
```

Comment:
jw-dock is inside jw-controls-right, so there is no reason to do the same transition on both. Just fade in jw-controls-right and everything inside fades too.

You could even make it simpler and just fade in jw-controls most of these are in jw-controls.